src/pages/ADN/main.ts

```diff
@@ -0,0 +1,86 @@
+import { pageInterface } from '../pageInterface';
+
+const style = require('!to-string-loader!css-loader!less-loader!./style.less');
+
+export const ADN: pageInterface = {
+  name: 'ADN',
+  domain: 'https://animationdigitalnetwork.fr/',
+  languages: ['French'],
+  type: 'anime',
+  isSyncPage(url) {
+    if ($('div[data-testid="video-player"]').length) return true;
+    return false;
+  },
+  isOverviewPage(url) {
+    if ($('div[data-testid="downline-content-test-id"]').length) return true;
+    return false;
+  },
+  sync: {
+    getTitle(url) {
+      return j.$('div[data-testid="player-content"] h1 a').text().trim();
+    },
+    getIdentifier(url) {
+      return utils.urlPart(url, 4) || '';
+    },
+    getOverviewUrl(url) {
+      return `${ADN.domain}/video/${ADN.sync.getIdentifier(url)}`;
+    },
+    getEpisode(url) {
+      return Number(utils.urlPart(url, 5) ? utils.urlPart(url, 5).split('-')[2] : '');
```

Comment:
probably better to use an regex like episode-(\d+)

Reply:
```suggestion
      return Number(utils.urlPart(url, 5).split('-')[2]);
```
Does this work for you?

Reply:
No using regex is better